function/make.go

```diff
@@ -227,5 +229,6 @@ var expressionType = reflect.TypeOf((*Expression)(nil)).Elem()
 var groupsType = reflect.TypeOf(Groups{})
 var contextType = reflect.TypeOf(EvaluationContext{})
 var timerangeType = reflect.TypeOf(api.Timerange{})
+var evaluationNotesType = reflect.TypeOf(EvaluationContext{})
```

Comment:
Seems weird that they ask for notes and get an `EvaluationContext `, why not `EvaluationNotes`?